site/app/libraries/Core.php

```diff
@@ -328,6 +338,23 @@ public function getSession(string $session_id, string $expected_user_id): bool {
         return true;
     }
 
+    /**
+     * Given an api_key (which should be coming from a parsed JWT), the database is queried to find
+     * a user id that matches the api key, and returning the user_id.
+     *
+     * @param string $api_key
+     *
+     * @return bool
+     */
+    public function getApiUser(string $api_key): bool {
```

Comment:
Let's call this `loadApiUser` as it doesn't return the User model, just a bool flag.